### DISCUSSION TITLE
HackMD介紹

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#954] [General]
因為最近開始在準備專題，發現了這個酷東西，完全就是一個為工程師們設計的線上共筆網站，而且還有超多功能和語法，真的超級有趣，分享給大家！下面的網址也有一些基本的介紹，有興趣的大家可以點進去探索

https://ithelp.ithome.com.tw/articles/10226420

**Comment by Zoe06060606** [comment]
我之前的課也用過這個 是真的很方便 再分享給別人筆記的時候還有整理筆記的時候都能很有系統

**Comment by ru0372572** [comment]
感謝分享~裡面有好多資源，很有系統，共筆看起來也好有親切感

**Comment by swerfg03joe** [comment]
個人感覺這個工具最實用的地方在繪製流程圖，以往要畫流程圖都是在PPT拉方塊，要花不少時間，如今只需要幾行code就可以完成

**Comment by Evan0979** [comment]
感謝分享,正好期末報告還沒有頭緒,之後可以參考看看

**Comment by dinosaur1009** [comment]
有點類似google文件，但是這個ㄉ功能更多，排版也更清楚！

**Comment by ruthchen1020** [comment]
我記得hackMD的創始人是台灣人喔!!

**Comment by WEI-CHEN-CHUN** [comment]
這個是一個很好的筆記整理，之前有看過學長用它來做類似紀錄的功能

**Comment by Valentinaa0214** [comment]
這個真的超好用的！！！
很美觀又支持多種格式及多種裝置的線上共編平台，
正值期末，拿來討論報告超好用的，
之前用過一次就覺得很驚艷，謝推薦

**Comment by Pietra1226** [comment]
每次查資料都會有他們網站的資料可以用，超級實用

**Comment by Twenty0522** [comment]
hackmd編輯非常方便，也有很可愛的設計，點進去發現原來也可以寫譜，超酷的！

**Comment by pierre0210** [comment]
hackmd 還可以畫[流程圖](https://flowchart.js.org/)跟做[簡報](https://hackmd.io/s/how-to-create-slide-deck) 這次期末專題的簡報就是用hackmd做的

**Comment by yuxinlin93** [comment]
hackmd的版面真的好好看 也很有組織性 這一次準備期末查資料的時候就看到很多是用hackmd紀錄的
